README.md

```diff
@@ -1,3 +1,112 @@
-# backend
+# iplayed-backend
 
-reservation system backend
+Welcome to iplayed-backend – the backend component of our exciting developer group's projects!
+
+Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+  - [Commiting](#commiting)
+- [Scripts](#scripts)
+- [Dependencies](#dependencies)
+- [Dev Dependencies](#dev-dependencies)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About
+
+iplayed-backend is the backend module for our innovative developer group's projects. We focus on integrating gamification into engaging activity venues like escape rooms, axe throwing, and paintball, creating a blend of technology and entertainment.
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, ensure you have the following prerequisites:
+
+- Node.js: Make sure you have Node.js installed. You can download it from here.
+
+### Installation
+
+1. [Clone the repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository):
+
+   `git clone https://github.com/iplayed/iplayed-backend.git`
+
+2. Ask on Discord for the `.env` file credentials if necessary and add it to your local copy.
+
+3. Keep the `.env-template` file and create a `.env` file with the credentials. To use the key from the credentials file, use ` const EXAMPLE_KEY = process.env.EXAMPLE_KEY;`
+
+4. Navigate to the project directory:
+
+   `cd iplayed-backend`
+
+5. Install dependencies:
+
+   `npm install`
+
+## Usage
+
+To run the development server, use the following command:
+
+`npm run dev`
+
+This will start the Express development server and allow you to preview your backend.
+
+## Commiting
+
+[Branch out](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) by following the structure [ISSUE-ID]-[Similar ticket title] and submit your [PR](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests). For example: `9-Login_form`
+
+After completing your component and you are ready to commit do the following...
+
+`npm run lint` and `npm run test`
+
+## Scripts
+
+- dev: Start the Express development server.
```

Comment:
this is the important part. Update from BACKEND

    "test": "jest --testTimeout=3000",
    "dev": "nodemon --exec node --experimental-specifier-resolution=node --loader ts-node/esm src/server.ts",
    "dev:seed": "ts-node --esm src/server.ts seed",
    "lint": "tsc && eslint --ext 'src/*.ts,*.json' --fix && prettier . --write",
    "prisma-update": "npx prisma db push",
    "prisma-generate": "npx prisma generate"